{
  "uid" : "d1203ec08b0f17ab",
  "name" : "登录测试反向用例",
  "fullName" : "cases.test_login.TestLogin#test_login_failed",
  "historyId" : "dbeb2a97f887c27b8015e9253b5aa614",
  "time" : {
    "start" : 1622204006545,
    "stop" : 1622204008579,
    "duration" : 2034
  },
  "status" : "broken",
  "statusMessage" : "KeyError: 'error'",
  "statusTrace" : "self = <cases.test_login.TestLogin object at 0x0000020D82000D30>\nhandler = (<appium.webdriver.webdriver.WebDriver (session=\"06356ee7-3934-4212-97e2-40d3008ad096\")>, <pages.page_action.PageAction object at 0x0000020D8200E828>)\ndata = {'error_msg': '账号或密码错误', 'name': '反向用例-用户名正确-密码错误', 'password': 'ipmnl', 'username': '18843010983'}\n\n    @allure.story('钓鱼人APP自动化测试反向用例测试')\n    @allure.title('登录测试反向用例')\n    @pytest.mark.parametrize('data', failed_data)\n    def test_login_failed(self, handler, data):\n        logger.info('开始执行反向测试用例：{}'.format(data['name']))\n        handler[1].login(data['username'], data['password'])\n        logger.info('反向用例输入完成')\n>       assert handler[1].is_toast_show(data['error']) == data['error']\nE       KeyError: 'error'\n\ncases\\test_login.py:48: KeyError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "handler",
    "time" : {
      "start" : 1622203974314,
      "stop" : 1622203993298,
      "duration" : 18984
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  }, {
    "name" : "prepare_to_login",
    "time" : {
      "start" : 1622203993298,
      "stop" : 1622204000625,
      "duration" : 7327
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "KeyError: 'error'",
    "statusTrace" : "self = <cases.test_login.TestLogin object at 0x0000020D82000D30>\nhandler = (<appium.webdriver.webdriver.WebDriver (session=\"06356ee7-3934-4212-97e2-40d3008ad096\")>, <pages.page_action.PageAction object at 0x0000020D8200E828>)\ndata = {'error_msg': '账号或密码错误', 'name': '反向用例-用户名正确-密码错误', 'password': 'ipmnl', 'username': '18843010983'}\n\n    @allure.story('钓鱼人APP自动化测试反向用例测试')\n    @allure.title('登录测试反向用例')\n    @pytest.mark.parametrize('data', failed_data)\n    def test_login_failed(self, handler, data):\n        logger.info('开始执行反向测试用例：{}'.format(data['name']))\n        handler[1].login(data['username'], data['password'])\n        logger.info('反向用例输入完成')\n>       assert handler[1].is_toast_show(data['error']) == data['error']\nE       KeyError: 'error'\n\ncases\\test_login.py:48: KeyError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "4042ffc4928d21ec",
      "name" : "log",
      "source" : "4042ffc4928d21ec.txt",
      "type" : "text/plain",
      "size" : 4399
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : true,
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 1
  },
  "afterStages" : [ {
    "name" : "handler::0",
    "time" : {
      "start" : 1622204008590,
      "stop" : 1622204009124,
      "duration" : 534
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "labels" : [ {
    "name" : "story",
    "value" : "钓鱼人APP自动化测试反向用例测试"
  }, {
    "name" : "epic",
    "value" : "钓鱼人APP自动化测试"
  }, {
    "name" : "feature",
    "value" : "钓鱼人APP自动化测试-登陆功能测试"
  }, {
    "name" : "tag",
    "value" : "@pytest.mark.usefixtures('prepare_to_login')"
  }, {
    "name" : "parentSuite",
    "value" : "cases"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "subSuite",
    "value" : "TestLogin"
  }, {
    "name" : "host",
    "value" : "DESKTOP-CIG6MR7"
  }, {
    "name" : "thread",
    "value" : "10336-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "cases.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "data",
    "value" : "{'name': '反向用例-用户名正确-密码错误', 'username': '18843010983', 'password': 'ipmnl', 'error_msg': '账号或密码错误'}"
  } ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ "@pytest.mark.usefixtures('prepare_to_login')" ]
  },
  "source" : "d1203ec08b0f17ab.json",
  "parameterValues" : [ "{'name': '反向用例-用户名正确-密码错误', 'username': '18843010983', 'password': 'ipmnl', 'error_msg': '账号或密码错误'}" ]
}